### PR TITLE
Move AdapterClientTestRun out of ClientTestsSuite

### PR DESCRIPTION
**Pull request by anwar6953**
This should resolve the failing test case: AdapterClientTestRun testAdapters.
The problem was that the test case did not get an opportunity to run its setup() method before the CDAP instance it was running against had started, due to it running within a test suite. Moving it out ensures that the test case can perform setup before the cdap instance starts.

This PR also introduces a deleteAll() method in ApplicationClient.

Build: https://builds.cask.co/browse/CDAP-RBT81-6/log

### CODE REVIEW

File: cdap-client/src/main/java/co/cask/cdap/client/ApplicationClient.java [hunk 1]
```diff
@@ -89,6 +89,17 @@ public void delete(String appId) throws ApplicationNotFoundException, IOExceptio
   }
 
   /**
+   * Deletes all applications.
+   *
+   * @throws IOException if a network error occurred
+   * @throws UnAuthorizedAccessTokenException if the request is not authorized successfully in the gateway server
+   */
+  public void deleteAll() throws IOException, UnAuthorizedAccessTokenException {
```
Comment (gokulavasan) [inline cdap-client/src/main/java/co/cask/cdap/client/ApplicationClient.java]: Do we need any testing for this method in ApplicationClientTestRun ?

**Comment by gokulavasan** [comment]
Comment about testing new deleteAll method. Rest looks good to me.

File: cdap-client-tests/src/test/java/co/cask/cdap/client/AdapterClientTest.java [hunk 2]
```diff
@@ -115,6 +120,8 @@ public void testAdapters() throws Exception {
 
     List<AdapterSpecification> finalList = adapterClient.list(namespaceId);
     Assert.assertEquals(0, finalList.size());
+
+    applicationClient.deleteAll();
```
Comment (gokulavasan) [inline cdap-client-tests/src/test/java/co/cask/cdap/client/AdapterClientTest.java]: Do we need to waitForDeleted here?

**Comment by gokulavasan** [comment]
LGTM :+1:

File: cdap-client/src/main/java/co/cask/cdap/client/ApplicationClient.java (hunk 1 above)
Comment (anwar6953) [inline cdap-client/src/main/java/co/cask/cdap/client/ApplicationClient.java]: added

File: cdap-client-tests/src/test/java/co/cask/cdap/client/AdapterClientTest.java (hunk 2 above)
Comment (anwar6953) [inline cdap-client-tests/src/test/java/co/cask/cdap/client/AdapterClientTest.java]: added

**Comment by gokulavasan** [comment]
LGTM :+1:

**Pull request merged**